### DISCUSSION TITLE
Telegram Applet

### DISCUSSION THREAD

**Discussion by alexeightsix** [i3/i3#5048] [Q&A] [answered]
hello!

I'm new to I3 and one thing I noticed right away is that I no longer see the telegram icon in the system tray. I use Telegram for work so need to see if I get any notifications... is it possible to add some sort of telegram applet to the system tray?

EDIT: opened telegram settings and toggled the show in system tray setting and it fixed the issue

**Comment by orestisfl** [comment, marked as answer]
Marking as answered
